src/lib.rs

```diff
@@ -314,6 +311,17 @@ where
 
         Ok(())
     }
+
+    fn clear(&mut self, color: Self::Color) -> Result<(), Self::Error> {
```

Comment:
Have you looked to see if there's any hardware clear in ST7735 that would be even faster than this?

Reply:
After perusing the docs, as far as I can see, there is no single command to clear the entire display (or a portion of it) after sending a given color only once. It seems like the only way to change the contents of the screen while its running is to first set a rectangular section and then write color values pixel by pixel over the SPI bus.